LibreNMS/Util/ObjectCache.php

```diff
@@ -53,6 +53,7 @@ public static function applications()
                 ->groupBy('app_type', 'app_instance')
                 ->orderBy('app_type')
```

Comment:
should sort by show_name or remove it, or try doing some natural sort trick in sql. Try benchmarking